starlette/routing.py

```diff
@@ -339,6 +339,7 @@ def __init__(
             self.app: ASGIApp = app
         else:
             self.app = Router(routes=routes)
+        self._routes_given = routes is not None
```

Comment:
What's the purpose of `_routes_given`? :thinking:

Reply:
Oops, that's a remainder from when I was experimenting with how best to pass the "we were given `Mount(routes=...)`" to the sessions middleware. 😄

Reply:
```suggestion
```